data_generate_for_gt.py

```diff
@@ -0,0 +1,147 @@
+import numpy as np
+from scipy.sparse import vstack, csr_matrix
+import gym, recogym
+import pandas as pd
+import math
+from copy import deepcopy
+from tqdm import tqdm, trange
+from recogym import env_1_args, Configuration
+from recogym.agents import RandomAgent, random_args
+import sys
+from multiprocessing import Pool
+
+def gen_data(data, num_products, va_ratio=0.2, te_ratio=0.2):
```

Comment:
Please extract out the common parts of these different files, so it is easier to read.
Also, I am not sure what does gt stands for ...

Reply:
all these scripts will be merged in next PR